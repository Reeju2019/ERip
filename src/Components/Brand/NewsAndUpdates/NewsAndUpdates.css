.newsPara{
  font-size: 14px;
}

.knowmore{
  color: blue;
  font-size: 16px;
  cursor: pointer;
}