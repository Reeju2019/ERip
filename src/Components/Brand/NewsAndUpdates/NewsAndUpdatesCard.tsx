import React from 'react'
import BrandPageData from '../../../Data/BrandPage.mock.json'
const NewsAndUpdatesCard: React.FunctionComponent = () => {
  return (
    <>
      <div className='mx-3 pb-3'>
        <div>
          <img src='static/brandPageImage/img-news.png' alt='' />
        </div>
        <h2 className='fs-4 my-2'>{BrandPageData.constData.newsAndUpdate.title}</h2>
        <p className='newsPara text-muted'>{BrandPageData.constData.newsAndUpdate.text}</p>
        <p className='knowmore'>{BrandPageData.constData.newsAndUpdate.knowMore}</p>
      </div>
    </>
  )
}

export default NewsAndUpdatesCard
