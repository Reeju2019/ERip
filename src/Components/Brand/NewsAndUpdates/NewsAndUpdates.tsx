import { Container, Row } from 'react-bootstrap'
import React from 'react'
import NewsAndUpdatesCard from './NewsAndUpdatesCard'
import './NewsAndUpdates.css'

const NewsAndUpdates: React.FunctionComponent = () => {
  return (
    <>
      <section className='pb-5'>
        <Container>
          <Row>
            <h3 className='mt-5 d-flex justify-content-center mb-3'>News & Updates</h3>
          </Row>
          <Row>
            <div>
              <NewsAndUpdatesCard />
              <NewsAndUpdatesCard />
              <NewsAndUpdatesCard />
              <NewsAndUpdatesCard />
            </div>
          </Row>
        </Container>
      </section>
    </>
  )
}

export default NewsAndUpdates
