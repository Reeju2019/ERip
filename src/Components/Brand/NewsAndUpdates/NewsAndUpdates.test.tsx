import { render, screen } from '@testing-library/react'
import renderer from 'react-test-renderer'
import ReactDOM from 'react-dom'
import NewsAndUpdates from './NewsAndUpdates'

test('news and updates', () => {
  render(<NewsAndUpdates />)
})

describe('App rendering specification', () => {
  it('SnapShot testing', () => {
    const component = renderer.create(<NewsAndUpdates />)
    const tree = component.toJSON()
    expect(tree).toMatchSnapshot()
  })
  it('render without crashing', () => {
    const div = document.createElement('div')
    render(<NewsAndUpdates />)
    ReactDOM.unmountComponentAtNode(div)
  })
})
