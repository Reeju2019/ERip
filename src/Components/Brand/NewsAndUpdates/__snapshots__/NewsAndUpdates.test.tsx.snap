// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App rendering specification SnapShot testing 1`] = `
<section
  className="pb-5"
>
  <div
    className="container"
  >
    <div
      className="row"
    >
      <h3
        className="mt-5 d-flex justify-content-center mb-3"
      >
        News & Updates
      </h3>
    </div>
    <div
      className="row"
    >
      <div>
        <div
          className="mx-3 pb-3"
        >
          <div>
            <img
              alt=""
              src="static/brandPageImage/img-news.png"
            />
          </div>
          <h2
            className="fs-4 my-2"
          >
            Quick & Eficent
          </h2>
          <p
            className="newsPara text-muted"
          >
            Lorem ipsum dolor sitali aliouiopsioi conseciu tetur uiyio adipiscing elitisyio sediuys dosuius eiusmod tempouisakiop siosuipil sincididuntis ut labore etios dio doloresio magna aliqua. Ut enim ad minim veniam, quis nostrud
          </p>
          <p
            className="knowmore"
          >
            Know More
          </p>
        </div>
        <div
          className="mx-3 pb-3"
        >
          <div>
            <img
              alt=""
              src="static/brandPageImage/img-news.png"
            />
          </div>
          <h2
            className="fs-4 my-2"
          >
            Quick & Eficent
          </h2>
          <p
            className="newsPara text-muted"
          >
            Lorem ipsum dolor sitali aliouiopsioi conseciu tetur uiyio adipiscing elitisyio sediuys dosuius eiusmod tempouisakiop siosuipil sincididuntis ut labore etios dio doloresio magna aliqua. Ut enim ad minim veniam, quis nostrud
          </p>
          <p
            className="knowmore"
          >
            Know More
          </p>
        </div>
        <div
          className="mx-3 pb-3"
        >
          <div>
            <img
              alt=""
              src="static/brandPageImage/img-news.png"
            />
          </div>
          <h2
            className="fs-4 my-2"
          >
            Quick & Eficent
          </h2>
          <p
            className="newsPara text-muted"
          >
            Lorem ipsum dolor sitali aliouiopsioi conseciu tetur uiyio adipiscing elitisyio sediuys dosuius eiusmod tempouisakiop siosuipil sincididuntis ut labore etios dio doloresio magna aliqua. Ut enim ad minim veniam, quis nostrud
          </p>
          <p
            className="knowmore"
          >
            Know More
          </p>
        </div>
        <div
          className="mx-3 pb-3"
        >
          <div>
            <img
              alt=""
              src="static/brandPageImage/img-news.png"
            />
          </div>
          <h2
            className="fs-4 my-2"
          >
            Quick & Eficent
          </h2>
          <p
            className="newsPara text-muted"
          >
            Lorem ipsum dolor sitali aliouiopsioi conseciu tetur uiyio adipiscing elitisyio sediuys dosuius eiusmod tempouisakiop siosuipil sincididuntis ut labore etios dio doloresio magna aliqua. Ut enim ad minim veniam, quis nostrud
          </p>
          <p
            className="knowmore"
          >
            Know More
          </p>
        </div>
      </div>
    </div>
  </div>
</section>
`;
